xen/arch/x86/slaunch.c

```diff
@@ -182,17 +208,43 @@ static struct slr_entry_policy *slr_get_policy(struct slr_table *slrt)
     return policy;
 }
 
-static void check_drtm_policy(struct slr_table *slrt,
-                              struct slr_entry_policy *policy,
-                              struct slr_policy_entry *policy_entry,
-                              const struct boot_info *bi)
+static void check_slrt_policy_entry(struct slr_policy_entry *policy_entry,
+                                    struct slr_table *slrt)
+{
+    if ( policy_entry->entity_type != SLR_ET_SLRT )
+        panic("Second entry of DRTM policy in SLRT is not SLRT: %#04x!\n",
```

Comment:
`SLRT is not SLRT` is a bit confusing. What does it mean?

Reply:
The first "SLRT" is related to DRTM policy which itself is stored within SLRT (`DRTM policy in SLRT`).  The policy has an entry describing measurement of SLRT.  I tried to avoid saying just "DRTM policy" as it provides too little context.  Simplified as this only harms the message.

https://github.com/TrenchBoot/xen/compare/f25f75bb48d3555bdc52df5afc2704763698046e..aa2fcc54193471e68cd2a730d5241e919128686b

Reply:
Looks much better, thanks.